main.nf

```diff
@@ -8,56 +8,85 @@
     Slack  : https://nfcore.slack.com/channels/genomeassembler
 ----------------------------------------------------------------------------------------
 */
-
-nextflow.enable.dsl = 2
+#!/usr/bin/env nextflow
```

Comment:
Given this is from an old template, you might be better off making a fresh pipeline with the new template.